youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -458,6 +458,9 @@ def add(meta_list, info_list=None):
         add('album_artist')
         add('disc', 'disc_number')
 
+        if info['filepath'][-3:] == 'mp4':
```

Comment:
1. Provide clear evidence this field is supposed to store the video URL.
2. This does not guarantee the actual container is `mp4`.
3. This won't work for audio.

Reply:
1. At this moment Podcast URL (PURL) field is used. What podcast has in common with video url? TV Network Name makes more sense, TV can interpreted as video source, like YouTube, and Network Name is URL. 
2. How to do it correctly? 
3. m4a? We need to extend this check?
Also, there is a good practice to use `-movflags +faststart` in mp4 files by default.

Reply:
At least it's an URL and not something completely unrelated. TV Network Name can't be interpreted as URL at all.

Reply:
https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/movenc.c#L3644
I think it's better to write url in inappropriate field, than not write at all

Reply:
Any progress on this?